### PR TITLE
[aws][eks] Enable deploying kubernetes-external-secrets

### PR DESCRIPTION
**Pull request by The-smooth-operator**

### CODE REVIEW

File: aws/eks/external_secrets.tf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+data "aws_iam_policy_document" "external_secrets" {
+  statement {
+    actions = [
+      "secretsmanager:GetSecretValue",
+    ]
+
+    resources = [
+      "arn:aws:secretsmanager:us-west-2:${data.aws_caller_identity.current.account_id}:secret:${var.secrets_path}",
```
Comment (The-smooth-operator) [inline aws/eks/external_secrets.tf:8]: @limed What would be the best way of getting secrets_path in here?

**Comment by The-smooth-operator** [comment]
@limed Let me know how this looks!

File: aws/eks/external_secrets.tf (hunk 1 above)
Comment (limed) [reply aws/eks/external_secrets.tf:8]: I think for now this is good, but maybe configure `${var.secrets_path}` as a list and create a loop through the paths for the variables.

Comment (The-smooth-operator) [reply aws/eks/external_secrets.tf]: thanks this makes sense

**Pull request merged**